tf2onnx/optimizer/transpose_optimizer.py

```diff
@@ -125,8 +136,8 @@ def _update_graph_nodes(self, nodes_to_extend, nodes_to_remove, has_input_change
         if nodes_to_remove:
             for n in nodes_to_remove:
                 ops.remove(n)
-        
-        if nodes_to_extend and len(nodes_to_extend) > 0:
+
+        if nodes_to_extend and nodes_to_extend:

```

Comment:
by intention here?

Reply:
yes, if nodes_to_extend  is the same as len(nodes_to_extend) != 0

Reply:
I mean, we should not have repeated nodes_to_extend. let me fix this in my local change.